core/domain/interaction_jobs_one_off.py

```diff
@@ -152,6 +152,43 @@ def reduce(key, values):
         yield (key, values)
 
 
+class MultipleItemInteractionLtOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that produces total number of all (exploration, state) pairs
+    that use the Multiple choice interaction or Item Selection
+    Interaction and total number of all (exploration, state) pairs
+    that use the Multiple choice interaction or Item selection
+    Interaction whose choice length is greater than 30.
+    """
```

Comment:
This is not true you do not return the exploration and state pairs. You return exploration ID and the length of the choice.

Reply:
Done